x-pack/plugins/maps/server/maps_telemetry/collectors/register.ts

```diff
@@ -31,6 +31,146 @@ export function registerMapsUsageCollector(
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        es_docs: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        es_point_to_point: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        es_tracks: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        kbn_region: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        kbn_tms_raster: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        ux_tms_mvt: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        ux_tms_raster: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        ux_wms: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+      },
+      scalingOptions: {
+        limit: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        clusters: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        mvt: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+      },
+      joins: {
+        term: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+      },
+      basemaps: {
+        auto: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        dark: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        roadmap: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
+        },
+        roadmap_desaturated: {
+          min: { type: 'long' },
+          max: { type: 'long' },
+          avg: { type: 'float' },
+          total: { type: 'long' },
```

Comment:
@thomasneirynck do you mind documenting the fields? You can do that by adding `_meta.description` like in:
```
{ type: 'long', _meta: { description: 'This field is for...' } }
```